bindings/rust/evmc-vm/src/container.rs

```diff
@@ -74,10 +89,48 @@ mod tests {
             set_option: None,
         };
 
+        let code = [0u8; 0];
```

Comment:
Maybe leave a note here that this is just for a dummy call.

Reply:
Well its a test, isn't it? Happy to add a comment just let me know what exactly.